### PR TITLE
Feat/indikator 1 dan 4

### PR DESCRIPTION
**Pull request by afifahfq**
Text finding for indicator 1 (changes from main)
Notes:
- checked for keywords in lvl 2 and lvl 4
- might remove lvl 5 if not needed
Example result for Batu Bara:
image

Text finding for indicator 4
Notes:
- checked for keywords in lvl 2, lvl 3, and lvl 4
- might remove lvl 5 if not needed
- used exclude words bcs keyword is too general
Example result for Batu Bara:
image

### CODE REVIEW

File: textfinding_indikator1.py
Comment (AlifahRBasyasya) [inline textfinding_indikator1.py:39]: Same as [#3 (comment)](https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/3#discussion_r1140977512)

File: textfinding_indikator1.py
Comment (AlifahRBasyasya) [inline textfinding_indikator1.py:28]: Same as [#3 (comment)](https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/3#discussion_r1140977512)

File: textfinding_indikator1.py
Comment (AlifahRBasyasya) [inline textfinding_indikator1.py:73]: I think it's better to replace the space with (`\s*`) to account for zero or multiple whitespaces, or you can also use (`convert_keywords`) function provided in utility, CMIIW!

CC: @akramab @AwwalaK

File: textfinding_indikator1.py
Comment (AlifahRBasyasya) [inline textfinding_indikator1.py:81]: Please use the (`convert_keywords`) function provided in `utility.py` to improve readability and decrease likelihood of typos and other minor mistakes

File: textfinding_indikator4.py
Comment (AlifahRBasyasya) [inline textfinding_indikator4.py:28]: Same as [#3 (comment)](https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/3#discussion_r1140977512)

File: textfinding_indikator4.py
Comment (AlifahRBasyasya) [inline textfinding_indikator4.py:43]: Same as [#3 (comment)](https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/3#discussion_r1140977512)

File: textfinding_indikator4.py
Comment (AlifahRBasyasya) [inline textfinding_indikator4.py:49]: Same as [#3 (comment)](https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/3#discussion_r1140977512)

File: textfinding_indikator4.py
Comment (AlifahRBasyasya) [inline textfinding_indikator4.py:56]: Please use the (`exclude_words`) function provided in `utility.py` to prevent redundancy

File: textfinding_indikator1.py
Comment (akramab) [inline textfinding_indikator1.py:43]: I think this `reg` variable here should be `key` instead?

File: textfinding_indikator4.py
Comment (AlifahRBasyasya) [inline textfinding_indikator4.py:63]: Please use the (`exclude_words`) function provided in `utility.py` to prevent redundancy

File: textfinding_indikator4.py
Comment (AlifahRBasyasya) [inline textfinding_indikator4.py:75]: Please remove this function and use the one provided in `utility.py` 🙏

File: textfinding_indikator1.py
Comment (akramab) [inline textfinding_indikator1.py:32]: Maybe we can modify every regex literal a lil bit to include `*` (asterisk) symbol so that it can handle cases with any number of whitespaces in between the key words. Just like here https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/7#discussion_r1141603066

File: textfinding_indikator1.py
Comment (afifahfq) [reply textfinding_indikator1.py:39]: already included validation if the key still exists in line 40 and using duplicate list using list(keyword) in line 39

File: textfinding_indikator1.py
Comment (afifahfq) [reply textfinding_indikator1.py:28]: already included validation if the key still exists in line 29 and using duplicate list using list(keyword) in line 28

File: textfinding_indikator1.py
Comment (afifahfq) [reply textfinding_indikator1.py:35]: already included validation if the key still exists in line 29

File: textfinding_indikator1.py
Comment (afifahfq) [reply textfinding_indikator1.py:47]: already included validation if the key still exists in line 40 and using duplicate list using list(keyword) in line 39

File: textfinding_indikator4.py
Comment (afifahfq) [reply textfinding_indikator4.py:28]: already using duplicate list using list(keyword) and included validation if the key still exists before removing

File: textfinding_indikator4.py
Comment (afifahfq) [reply textfinding_indikator4.py:43]: already using duplicate list using list(keyword) and included validation if the key still exists before removing

File: textfinding_indikator4.py
Comment (afifahfq) [reply textfinding_indikator4.py:40]: already using duplicate list using list(keyword) and included validation if the key still exists before removing

File: textfinding_indikator4.py
Comment (afifahfq) [reply textfinding_indikator4.py:49]: already using duplicate list using list(keyword) and included validation if the key still exists before removing

File: textfinding_indikator1.py
Comment (afifahfq) [reply textfinding_indikator1.py]: I've fixed it according to the suggested changes. Please give it another review if you've time. Thank you!

File: textfinding_indikator1.py
Comment (afifahfq) [reply textfinding_indikator1.py]: Thanks for pointing it out! I've fixed it according to the suggested changes. Please give it another review if you've time.

**Review by AlifahRBasyasya — APPROVED** [review]
LGTM! Thanks for the changes!

File: textfinding_indikator1.py
Comment (akramab) [inline textfinding_indikator1.py:52]: I am just curious, is the forward slash `/` between the words arsitektur and spbe intended?

File: textfinding_indikator1.py
Comment (akramab) [inline textfinding_indikator1.py:18]: I think there's a little typo here. This one initialize `init_keywords` variable (with `s`), but `init_keyword` (without `s`) is the one that's being used throughout the file

**Review by akramab — COMMENTED** [review]
All's good, just need to fix a lil typo before I give my approval :D

File: textfinding_indikator1.py
Comment (afifahfq) [reply textfinding_indikator1.py]: Thanks for the keen eye! I've fixed it according to the suggested changes. Please give it another review if you've time.

**Review by akramab — APPROVED** [review]
LGTM, everything's good, thanks for the changes! Just a little note about the `tempCodeRunnerFile.py`. Maybe you could exclude that file from the commit (?)

**Review by AwwalaK — APPROVED** [review]

**Pull request merged**
